modules/installing-gcp-cluster-creation.adoc

```diff
@@ -5,57 +5,41 @@
 [id="installing-gcp-cluster-creation_{context}"]
 = Configuring user-defined labels and tags for GCP
 
+Configuring user-defined labels and tags for {gcp-full} means that you can apply key-value pairs to your cloud resources for the purposes of organizing, managing, and automating your infrastructure.
+
 .Prerequisites
 
 * The installation program requires that a service account includes a `TagUser` role, so that the program can create the {product-title} cluster with defined tags at both organization and project levels.
 
 .Procedure
 
-* Update the `install-config.yaml` file to define the list of desired labels and tags.
+* Update the `install-config.yaml` file to define the list of required labels and tags.
 +
 [NOTE]
 ====
-Labels and tags are defined during the `install-config.yaml` creation phase, and cannot be modified or updated with new labels and tags after cluster creation.
+If you set labels and tags during creation of the `install-config.yaml` configuration file, you cannot create new or change existing labels and tags after creation of the cluster.
```

Comment:
One minor suggestion, replace "change existing" with "update existing", for the sake of consistency, as we use the words "**create**", "**get**", "**update**", "**delete**" for resources provisioning. Thanks!